1.0.0_lint_schema.json

```diff
@@ -0,0 +1,176 @@
+                },
+                "licenses": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/Copyright"
+                    }
+                }
+            },
+            "required": ["copyrights", "licenses", "path"],
+            "title": "File"
+        },
+        "Copyright": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "value": {
+                    "type": "string"
+                },
+                "source": {
+                    "type": "string"
+                }
+            },
+            "required": ["source", "value"],
+            "title": "Copyright"
+        },
+        "NonCompliant": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "missing_licenses": {
+                    "$ref": "#/definitions/MissingLicenses"
+                },
+                "unused_licenses": {
+                    "type": "array",
+                    "items": {}
+                },
+                "deprecated_licenses": {
+                    "type": "array",
+                    "items": {}
+                },
+                "bad_licenses": {
+                    "$ref": "#/definitions/BadLicenses"
+                },
+                "licenses_without_extension": {
+                    "$ref": "#/definitions/LicensesWithoutExtension"
+                },
+                "missing_copyright_info": {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "missing_licensing_info": {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "read_errors": {
+                    "type": "array",
+                    "items": {}
+                }
+            },
+            "required": [
+                "bad_licenses",
+                "deprecated_licenses",
+                "licenses_without_extension",
+                "missing_copyright_info",
+                "missing_licenses",
+                "missing_licensing_info",
+                "read_errors",
+                "unused_licenses"
+            ],
+            "title": "NonCompliant"
+        },
+        "BadLicenses": {
+            "type": "object",
+            "additionalProperties": { "type": "object" },
+            "title": "BadLicenses"
+        },
+        "LicensesWithoutExtension": {
+            "type": "object",
+            "additionalProperties": { "type": "object" },
+            "title": "LicensesWithoutExtension"
+        },
+        "MissingLicenses": {
+            "type": "object",
+            "additionalProperties": { "type": "object" },
+            "title": "MissingLicenses"
+        },
```

Comment:
If I understand it correctly these objects are not defined or explicitly structured.

Reply:
Yep, you're right. That's because the key of these objects is in a sense arbitrary, i.e. the license name. I don't know JSON schema well enough to know whether this can be alleviated. I think for a first version, this is fine though.

Reply:
Least we could do is force the keys to be strings https://stackoverflow.com/a/26848488